templates/docs/examples/brochure/hero.html

```diff
@@ -0,0 +1,28 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Hero{% endblock %}
+
+{% block style %}
+{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+
+<div class="p-hero">
+    <div class="p-hero__header">
+        <h1>Company culture</h1>
+        <p class="p-heading--2">We believe that talent is evenly distributed around the world. Diversity is part of our strength. What unifies us isn’t our background, it’s our mission to amplify open source.</h2>
+    </div>
+    <div class="p-hero__content">
+        <p>The future is already here, as open source. Canonical delivers it to the world. We play a critical role in broadening the benefits of open source to more people and more industries than ever before. This means bringing new work opportunities to people regardless of their location too.
+        </p>
+        <p>At Canonical, we are proud to provide equal opportunities to everyone regardless of gender, sexual orientation, race, ethnicity, religion, veteran status, disability or any other protected characteristic.
+        </p>
+    </div>
+    <div class="p-hero__footer">
+        <hr/>
```

Comment:
If footer always has a border then maybe it should be coded into the football element.

Reply:
It doesn't …